src/ostree/ot-builtin-checkout.c

```diff
@@ -145,12 +148,18 @@ process_one_checkout (OstreeRepo *repo, const char *resolved_commit, const char
                              || opt_process_passthrough_whiteouts;
 
   /* If we're doing composefs, then this is it */
-  if (opt_composefs)
+  if (opt_composefs || opt_composefs_noverity)
     {
       if (new_options_set)
         return glnx_throw (error, "Specified options are incompatible with --composefs");
-      return ostree_repo_checkout_composefs (repo, NULL, AT_FDCWD, destination, resolved_commit,
-                                             cancellable, error);
+      g_autoptr (GVariantBuilder) checkout_opts_builder
```

Comment:
Same comments as above: stack-allocated goes here better.  Or `g_variant_new_parsed()`.  And be careful about the floating return value of `_end()`.

Reply:
Switched to stack allocations